homepage/homepage/content/docs/server-side/runtime.mdx

```diff
@@ -0,0 +1,59 @@
+
+import { CodeGroup } from "@/components/forMdx";
+export const metadata = {
+    description: "How to support different runtimes as Cloudflare and Vercel Edge"
+};
+
+# Wasm on Edge runtimes
+
+On some edge platforms, such as Cloudflare Workers or Vercel Edge Functions, WASM may fail to load due to environment security restrictions. To prevent this, you should import the WASM loader before using Jazz. Otherwise, Jazz will fall back to the slower JavaScript implementation. For example:
```

Comment:
```suggestion
To maximise compatibility, Jazz falls back to a slower, JavaScript crypto implementation if the faster WASM implementation is not available.

On some edge platforms, such as Cloudflare Workers or Vercel Edge Functions, environment security restrictions may trigger this fallback unnecessarily.

You can ensure that Jazz uses the faster WASM implementation by importing the WASM loader before using Jazz. For example:
```

We can focus on a positive story here with a few minor wording changes :)

Reply:
I love it :+1: